common/utilities/tests/test_logger.py

```diff
@@ -0,0 +1,124 @@
+import io
+import logging
+from unittest import TestCase
+from unittest.mock import patch, MagicMock
+from common.utilities import integration_adapters_logger as log
+
+
+class TestLogger(TestCase):
+
+    def test_dictionary_formatting(self):
+        # Tests both removing the spaces and surrounding values with quotes if needed
+        input_dict = {
+            'Key With Space': 'value with space',  # Needs quotes
+            'EasyKey': 'EasyValue',  # Doesn't need quotes
+        }
+
+        expected_output = {
+            'Key With Space': 'KeyWithSpace="value with space"',
+            'EasyKey': 'EasyKey=EasyValue'
+        }
+        output = log._format_values_in_map(input_dict)
+        self.assertEqual(output, expected_output)
+
+    @patch('sys.stdout', new_callable=io.StringIO)
+    def test_audit(self, mock_std):
+        log.load_global_log_config()
```

Comment:
In a few of these tests, you call this function. And each time you do, you are adding another handler to the root logger. I mean, the tests are all passing (after fixing the import issue), but it doesn't feel clean/proper.

Reply:
Yeah I had an issue with this actually, I wanted to run it once before all but I ran into issues with the mocking because adding it before the test seems to not mock stdout as expected, running it at the start of each test was the only consistent solution. Other ideas would be good if you have any